cypress/e2e/chris.cy.ts

```diff
@@ -0,0 +1,39 @@
+const xml2js = require('xml2js')
+
+describe("", () => {
+    it("Obstacle #72946", () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72946');
+        // download the xml file
+        cy.request({
+            url: '/Obstacles/GetObstacleFile?obstacleId=72946&filename=Catalog.xml',
+            method: 'GET',
+            encoding: 'binary',
+          }).then((response) => {
+            const fileName = 'Catalog.xml';
```

Comment:
Do you need to write response into a local file then read it again?
I feel you can do it directly

Reply:
Update code without reading and writing files.